File: app/src/main/java/com/iita/akilimo/interfaces/IVolleyDeleteCallback.java
package com.iita.akilimo.interfaces;

import com.android.volley.VolleyError;

/**
 * Interface for volley actions callbacks
 */
public interface IVolleyDeleteCallback {
    /**
     * @param result
     */
    void onDeleted(String result);

    /**
     * @param error
     */
    void onError(VolleyError error);
}

File: app/src/main/java/com/iita/akilimo/interfaces/IVolleyDeleteCallback.kt
package com.iita.akilimo.interfaces

import com.android.volley.VolleyError

/**
 * Interface for volley actions callbacks
 */
interface IVolleyDeleteCallback {
    /**
     * @param result
     */
    fun onDeleted(result: String)

    /**
     * @param error
     */
    fun onError(error: VolleyError)
}
